test/clj/rems/api/testing.clj

```diff
@@ -146,6 +146,10 @@
   (assert-response-is-ok response)
   (read-body response))
 
+(defn read-unauthorized-body [response]
```

Comment:
This is seems like a useful little helper, but please fix the name. It doesn't have anything to do with unauthorized does it? How about a name like `status-and-body` or `read-body-and-status`?

Reply:
Yes, that is true, I was naming it based on purpose, not the action, my bad

Reply:
addressed